modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/entry/ViewEntryUpperToolbar.es.js

```diff
@@ -31,6 +31,7 @@ function ViewEntryUpperToolbar({
 	dataRecordId,
 	history,
 	page,
+	showButtonsOptions,
```

Comment:
Hey @naraandrade,

Could you please rename it to something like showButtons or buttonsOptions?

Thanks!